demos/demo-minimal-js/index.html

```diff
@@ -1,4 +1,5 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
We could use UTF-8 character encoding as well

Reply:
Added!